src/main/java/Duke.java

```diff
@@ -1,10 +1,149 @@
+import java.util.Scanner;
+
+
 public class Duke {
+
+    public static int listSize = 0;
+    public static Task[] taskList = new Task[100];
+
+    public static void addTask(Task task) {
+        taskList[listSize] = task;
+        listSize++;
+        System.out.printf("Got it. I've added this task:\n%s\nNow you have %d tasks in the list.\n",
+                task.print(), listSize);
+    }
+
+    public  static  void todo(String line, String[] words) {
+        if (words.length >= 2) {
+            String task = line.substring(line.indexOf(' ') + 1);
+            Todo todo = new Todo(task);
+            addTask(todo);
+        } else {
+            System.out.println("Usage: todo <task>");
+        }
+    }
+
+    public static void deadline(String line) {
+        try {
+            String task = line.substring(line.indexOf(' ')+1, line.indexOf("/by")-1);
+            String date = line.substring(line.indexOf("/by")+4);
+            Deadline deadline = new Deadline(task, date);
+            addTask(deadline);
+        } catch (StringIndexOutOfBoundsException ex) {
+            System.out.println("Usage: deadline <task> /by <date>");
+        }
+    }
```

Comment:
Maybe you can consider using a verb for methods instead of a noun?
 eg makeDateline